packages/httpsnippet-client-api/src/index.ts

```diff
@@ -47,10 +47,10 @@ function buildAuthSnippet(authKey: string | string[]) {
       auth.push(`'${token.replace(/'/g, "\\'")}'`);
     });
 
-    return `sdk.auth(${auth.join(', ')});`;
+    return `${sdkVariable}.auth(${auth.join(', ')});`;
```

Comment:
found a bug in these `.auth()` snippets with custom variables while building this out.